### ISSUE TITLE
Optimisation report causing build failure when using Intel oneAPI

### ISSUE THREAD

**Issue by branfosj** [DrTimothyAldenDavis/GraphBLAS#211]
For an build using the Intel compilers (Intel oneAPI) we've seen several failures caused by the `-qopt-report=5` option leading to high memory usage (120GB) causing the build to be killed due to lack of memory.

Errors look like:
```
icc: error #10106: Fatal error in /apps/Test2/software/intel-compilers/2021.2.0/compiler/2021.2.0/linux/bin/intel64/../../bin/intel64/mcpcom, terminated by kill signal
compilation aborted for /dev/shm/SuiteSparse/5.10.1/intel-2021a-METIS-5.1.0/SuiteSparse-5.10.1/GraphBLAS/Source/GB_AxB_saxpy_generic.c (code 1)
```

We've seen this with various versions of SuiteSparse and various versions of the Intel OneAPI compilers. The failure is always while building GraphBLAS (hence reporting it here and not in the SuiteSparse repository). If we disable the generation of that report and the build is fine.

So, is there a good reason to keep generating that report by default? Or, instead, could the default be a lower level?

**Comment by DrTimothyAldenDavis** [comment]
I can turn it off by default.  I'll comment out the line in the cmake that sets those options. I had it turned on when looking at the performance of GraphBLAS on the Intel devcloud, but there's no need to leave it on in production.

**Comment by DrTimothyAldenDavis** [comment]
Fixed in v8.0.2.
